tcf_website/management/commands/load_grades.py

```diff
@@ -235,11 +244,11 @@ def load_dict_into_models(self):
                 'subdepartment': row[0],
                 'number': row[1],
                 'first_name': row[2],
-                'middle_name': row[3],
-                'last_name': row[4],
-                'email': row[5],
+                'middle_name': '',  # for backward compatibility
```

Comment:
Was middle name removed from the loading because recent files don't include it anymore?

Reply:
No, because 1) there are inconsistencies in the CSV files and 2) `Instructor` table doesn't have middle name, so I figured we could drop it entirely.

Reply:
I think that's fair